packages/Molecules/Layout/PageLayout.vue

```diff
@@ -59,6 +60,9 @@ export default {
       }
 
       return {}
+    },
+    bgGraphic () {
+      return this.backgroundGraphic ? { 'background-image': `url(${this.backgroundGraphic})` } : null
```

Comment:
Hi @alan-cole I feel this logic is not needed, because background image will cover the graphic, am I right?
That can save make everything on it's own, not have coupled logic.

Reply:
If we do this, then a background graphic will need to be defined for every use of PageLayout. (Otherwise we get this)
![screen shot 2018-09-03 at 11 39 59 am](https://user-images.githubusercontent.com/12739575/44963480-2a8dce80-af6e-11e8-8afe-abbffef36a2a.png)

Personally, I think it's a better solution if you can choose to use a graphic or not, rather than force the use of a graphic (or risk broken HTML), but this is your call - I will implement as requested.